docs/reference/plugins/developing_custom_plugins.md

```diff
@@ -0,0 +1,54 @@
+# Developing Custom Plugins
+
+Shift Plugins are developed as custom *PySide2* classes inheriting from `QtWidgets.QWidget`.
+
+## Writing a Plugin Class
+
+Shift's native plugins code, together with a template file can be found in the `shift/plugins` folder. These should serve as a reference for implementing your custom logic and integrating it correctly inside Shift.
```

Comment:
I would move this after paragraph in line 9

Reply:
done